src/subscription_manager/scripts/rhsm_d.py

```diff
@@ -96,24 +96,38 @@ def excepthook_logging(exc_type, exc_value, exc_traceback):
 
 conf = config.Config(get_config_parser())
 
-enable_debug = False
-
 
 def debug(msg):
+    """
+    Print debug messages to console or rhsm.log
+    :param msg:
+    :return:
+    """
     if enable_debug:
         log.debug(msg)
         print(msg)
+    else:
+        log.debug(msg)
 
 
 def in_warning_period(sorter):
-
+    """
+    Is any entitlement certificate expired?
+    :param sorter:
+    :return:
+    """
     for entitlement in sorter.valid_entitlement_certs:
         if entitlement.is_expiring():
             return True
     return False
 
 
 def pre_check_status(force_signal):
+    """
+    Pre check status of the system. Is it registered against classic. Is it registered?
+    :param force_signal: When this is not None, then this value is returned and this signal si
```

Comment:
I think there might have been more to this comment. "... this signal is sent"  perhaps?